client/src/pages/Home.js

```diff
@@ -75,6 +76,13 @@ export function Home() {
 		setIsUploadFailure(true);
 	};
 
+	const showOnClientInfoClick = (i) => {
```

Comment:
```suggestion
	const handleRowClick = (i) => {
```
`show` in the name of the function suggests that it only shows, but it also hides so it's misleading. We can give it a more generic name like `handleRowClick`—what do you think?

Reply:
I think that is a great and general name, thank you so much.